Mathlib/Analysis/SpecialFunctions/Complex/Analytic.lean

```diff
@@ -0,0 +1,56 @@
+/-
+Copyright (c) 2024 Geoffrey Irving. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Geoffrey Irving
+-/
+import Mathlib.Analysis.Analytic.Composition
+import Mathlib.Analysis.Analytic.Constructions
+import Mathlib.Analysis.Complex.CauchyIntegral
+import Mathlib.Analysis.SpecialFunctions.Complex.LogDeriv
+
+/-!
+# Various complex special functions are analytic
+
+`exp`, `log`, and `cpow` are analytic, since they are differentiable.
+-/
+
+open Complex Set
+open scoped Topology
+
+variable {E : Type} [NormedAddCommGroup E] [NormedSpace ℂ E]
+variable {f g : E → ℂ} {z : ℂ} {x : E} {s : Set E}
+
+/-- `exp` is entire -/
+theorem AnalyticOn.cexp : AnalyticOn ℂ exp univ := by
+  rw [analyticOn_univ_iff_differentiable]; exact differentiable_exp
+
+/-- `exp` is analytic at any point -/
+theorem AnalyticAt.cexp : AnalyticAt ℂ exp z :=
```

Comment:
could you instead call this `analyticAt_cexp` and have a theorem `AnalyticAt.cexp` for dot notation, similar to `AnalyticAt.clog` below?

Reply:
Oops, fixed.  Thanks!